packages/apify/src/actor.ts

```diff
@@ -836,9 +836,27 @@ export class Actor<Data extends Dictionary = Dictionary> {
     }
 
     /**
-     * Returns a new {@apilink ApifyEnv} object which contains information parsed from all the `APIFY_XXX` environment variables.
+     * Modifies Actor env vars so parsing respects the structure of {@apilink ApifyEnv} interface.
+     */
+    private getModifiedActorEnvVars = () => {
```

Comment:
i'd keep this as a regular method instead of function expression, to be consistent - we dont use this anywhere (unless really needed)